src/components/DomEventsTable.js

```diff
@@ -0,0 +1,204 @@
+import React, { useRef, useState, useCallback } from 'react';
+import IconButton from './IconButton';
+import TrashcanIcon from './icons/TrashcanIcon';
+import EmptyStreetImg from '../images/EmptyStreetImg';
+import StickyList from './StickyList';
+import { VirtualScrollable } from './Scrollable';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import CopyButton from './CopyButton';
+import Select from 'react-select';
+
+const defaultNamesOptions = [
```

Comment:
By default, I think these options have to be filtered **out** instead of being selected (as per https://github.com/testing-library/testing-playground/issues/171)

@smeijer can you please correct me if I'm wrong

Reply:
Correct @sumeesh879. Thanks for the review 👍

Reply:
Thanks @sumeesh879, I completely misunderstood!
@smeijer: according to you should the user be able to select these options or are these completely disabled?

Reply:
All event types should be selectable (and selected), but the ones that I mention, should be deselected by default. Because they generate a lot of noise.

So not disabled (unavailable). The user just needs to select them explicitly.

Reply:
I replaced the code to make the noisy events excluded by default.